test/passing/doc_comments.mli

```diff
@@ -180,7 +201,7 @@ val x : x
 
 (** [a]'c [b]'s [c]'c *)
 
-(** return true if {%\gamma(lhs) \subseteq \gamma(rhs)%} *)
+(** return true if [\gamma(lhs) \subseteq \gamma(rhs)] *)
```

Comment:
The `{%` `%}` markup ought to be preserved, right?

Reply:
In this case (and the next one) Odoc gives a warning and we could refuse to format with an error.

Reply:
Oh. So odoc doesn't support the whole ocamldoc language? `{%` and `%}` are documented markup.

Reply:
Odoc gives a warning and turn it into a code span:
```
'{%...%}' (raw markup) needs a target language.
Suggestion: try '{%html:...%}'.
```
Warnings are easily handled, I did not code this part yet.
In case of warning, do you think we should print the warning and let the doc comment as is or refuse to format the whole file ?
The first seems more reasonable but without a `-Werror` flag (or maybe when `--check` is set), it could stay unnoticed.

Reply:
This seems like an odoc bug / divergence from spec to me. The `latex` language is the default: http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sss%3Atarget-specific-syntax

Reply:
This is now fixed.